alphapulldown/utils/modelling_setup.py

```diff
@@ -35,25 +35,31 @@ def parse_fold(input, features_directory, protein_delimiter):
         for protein_fold in protein_folds:
             name, number, region = None, 1, "all"
 
-            match len(protein_fold):
-                case 1:
-                    name = protein_fold[0]
-                case 2:
-                    name, number = protein_fold[0], protein_fold[1]
-                    if ("-") in protein_fold[1]:
+            if len(protein_fold) ==1:
+                # protein_fold is in this format: [protein_name]
+                name = protein_fold[0]
+            elif len(protein_fold) > 1:
```

Comment:
@maurerv do you think an else...raise is necessary here if ```len(protein_fold)```is 0 due to mistakes by the user?